### PR TITLE
[CHAOS-1459] Update SMP 0.8.0 docs for CE 

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update SMP docs for CE

For reviewers, previews are available:

https://64c079e700ce440b7ac41653--harness-developer.netlify.app/docs/self-managed-enterprise-edition/introduction/getting-started-with-self-managed-enterprise-edition

https://64c079e700ce440b7ac41653--harness-developer.netlify.app/docs/self-managed-enterprise-edition/introduction/harness-self-managed-enterprise-edition-overview

https://64c2687c3ef89b00859a7d63--harness-developer.netlify.app/docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations#deployment-infrastructure

https://64c079e700ce440b7ac41653--harness-developer.netlify.app/docs/self-managed-enterprise-edition/self-managed-helm-based-install/harness-helm-chart

https://64c079e700ce440b7ac41653--harness-developer.netlify.app/docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-harness-self-managed-enterprise-edition-using-helm-ga

https://64c079e700ce440b7ac41653--harness-developer.netlify.app/docs/self-managed-enterprise-edition/self-managed-helm-based-install/install-in-an-air-gapped-environment

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ This topic lists the supported features and integrations you can use to install
    | Security Testing Orchestration | 4 | 3| 7 | 0 |
    | Feature Flags | 3 | 3 | 6 | 0 |
    | Service Reliability Management | 6 | 8 | 18 | 0 |
+   | Chaos Engineering | 5 | 4 | 8 | 20 |
```
Comment (sagarkrsd) [inline docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md:21]: Seems this should be the prod environment resource requirements, right?

Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md]: @sagarkrsd Yes. Nice catch, thank you. Updated.

**Review by sagarkrsd — APPROVED** [review]
Thanks @brian-f-harness 🙌 LGTM!

**Review by pratmit — APPROVED** [review]

**Review by aimurphy — APPROVED** [review]
lgtm

**Pull request merged**